app/models/models.go

```diff
@@ -153,6 +153,62 @@ func (r *ListQueueRequest) SetAttributesFromForm(values url.Values) {
 	r.QueueNamePrefix = values.Get("QueueNamePrefix")
 }
 
+/*** Send Message Request */
+
+func NewSendMessageRequest() *SendMessageRequest {
+	return &SendMessageRequest{
+		MessageAttributes:       make(map[string]MessageAttributeValue),
+		MessageSystemAttributes: make(map[string]MessageAttributeValue),
+	}
+}
+
+type SendMessageRequest struct {
+	DelaySeconds            int                              `json:"Del1aySeconds" schema:"DelaySeconds"`
+	MessageAttributes       map[string]MessageAttributeValue `json:"MessageAttributes" schema:"MessageAttributes"`
```

Comment:
Yes, `MessageAttributes` and `MessageSystemAttributes` are the ability to append any fields on users demand.
And sorry, I haven't checked the names for `MessageSystemAttributes` is defined by AWS. 

How do you think to mark `MessageSystemAttributes` as "unsupported" with definition of current `map[string]MessageAttributeValue` at this time? 
Because AFAIK this "goaws" is not support emulation of AWS X-Ray. 
We can also add support later if we or users find a MessageSystemAttribute name between SQS/SNS.

Reply:
Sorry to miss the response 🙇 
I think we don't have to change `map[string]` for `MessageSystemAttributes` soon since the latest AWS SDK accept the `map[string]` for it.
<img width="865" alt="image" src="https://github.com/Admiral-Piett/goaws/assets/156628139/8fb20dab-0b0d-4d38-8e70-2e12df756d8b">

I'll mark it as TODO.

Reply:
Yeah you can leave it for now.  I think it would be easier to manage long term with a nested struct, but I can deal with it later.  Thanks anyway!